src/main/java/net/masterthought/cucumber/json/deserializers/StatusDeserializer.java

```diff
@@ -25,7 +26,7 @@ public Status deserialize(JsonNode rootNode, Configuration configuration) {
         if (UNKNOWN_STATUSES.contains(status)) {
             return Status.UNDEFINED;
         } else {
-            return Status.valueOf(status.toUpperCase());
+            return Status.valueOf(status.toUpperCase(Locale.US));
```

Comment:
Modify sample.json so there is at least one status for which this fix is applied

Reply:
sample.json already have entries with "status":"failed" so no change is necessary at the moment.